tests/foreman/rhai/conftest.py

```diff
@@ -6,22 +6,28 @@
 from fauxfactory import gen_string
 from requests.exceptions import HTTPError
 
-from robottelo.constants import DEFAULT_ORG
+from robottelo import manifests
+from robottelo.api.utils import upload_manifest
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import DISTRO_RHEL7
+from robottelo.constants import DISTRO_RHEL8
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.vm import VirtualMachine
+
 
 LOGGER = logging.getLogger('robottelo')
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
```

Comment:
Our code standards still prefer single quotes over double quotes. There are a few instances of this conversion in this pull request.

Reply:
Done in 549ce233f13051aac059b904ed75ffdce4d16f17